serverless.yml

```diff
@@ -13,6 +13,9 @@ provider:
   stackTags:
     "wma:applicationId": "Aquarius TS Capture Error Handler"
     "wma:contact": "Andrew Yan ayan@usgs.gov"
+    "wma:role": etl
+    "wma:system": NWIS
+    "wma:subSystem": NWISWeb-Capture
```

Comment:
Looks good.  Is there a list of valid values for new developers?  Is it somewhat free form as to the
allowed values, for instance, for  "wma:role"?

Reply:
I'm not sure if there's a list that our overlords maintain. If there is, I don't know about it. We should just use values that make our lives easier and use them consistently.